packages/ui/components/ui/gradient-header.tsx

```diff
@@ -0,0 +1,8 @@
+/**
+ * A header with text whose color is a gradient of brand colors.
+ */
+export const GradientHeader = ({ children }: { children: string }) => (
```

Comment:
![image](https://github.com/penumbra-zone/web/assets/1121544/55661dbe-db60-43f0-afbe-b1fa6683b3f8)

Reply:
Nice abstracting this out